packages/@lwc/integration-karma/test/signal/reactivity/index.spec.js

```diff
@@ -0,0 +1,105 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+
+import Reactive from 'x/reactive';
+import NonReactive from 'x/nonReactive';
+import ExplicitSubscribe from 'x/explicitSubscribe';
+import List from 'x/list';
+
+import { Signal } from 'x/signal';
```

Comment:
It's a little weird to use LWC module resolution for this. I see why you're doing it, because to make it easier to import elsewhere in the test collateral. Maybe just leave a comment here, since at first glance it looks like you're importing a component named `x-signal`.